tests/gnrc_tcp/tests/test_02-conn_lifecycle_as_server.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2018   Simon Brummer <simon.brummer@posteo.de>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+import socket
+import threading
+
+from shared_func import getRiotLinkLocalAddr
+
+HOST_IF = 'tap0'
+SERVER_PORT = 15000
+
+def tcpClient(serverAddr):
+    sock = socket.socket(socket.AF_INET6, socket.SOCK_STREAM)
+    sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+
+    addrInfo = socket.getaddrinfo(serverAddr + '%' + HOST_IF, SERVER_PORT, type = socket.SOCK_STREAM)
+
+    sock.connect(addrInfo[0][-1])
+    sock.close()
+
+def testfunc(child):
+    clientHandle = threading.Thread(target = tcpClient, args = (getRiotLinkLocalAddr(child),))
+
+    # Setup RIOT Node wait for incomming connections from Hostsystem
+    child.sendline('gnrc_tcp_tcb_init')
+    child.sendline('gnrc_tcp_open_passive AF_INET6 ' + str(SERVER_PORT))
+
+    clientHandle.start()
+    child.expect_exact('gnrc_tcp_open_passive: returns 0')
```

Comment:
I checked with `echo=True` and this line never comes (nor any similar line).

Reply:
Well I am able to see it. If you try again It should work too. The test was not able to deal with tapbr last week.